common.js

```diff
@@ -24,7 +24,7 @@ const DEFAULT_OPTIONS = {
 };
 
 async function gettingOptions() {
-  const options = await browser.storage.sync.get(null);
+  options = await browser.storage.sync.get(null);
```

Comment:
It would be better to use `let`, rather than implicit local variable.

```suggestion
  let options = await browser.storage.sync.get(null);
```